examples/open-loop/plot_loop.py

```diff
@@ -33,18 +30,18 @@ def set_style(self: Parser):
         plt.rcParams.update({'font.weight': "regular"})
         plt.rcParams.update({'axes.linewidth': 1})
 
-    def segmentize(self: Parser, key: str) -> np.ndarray:
+    def segmentize(self, key: str) -> np.ndarray:
         """
         Segmentizes the data log based on a key signal, such as a trigger.
         """
 
         segments = []
         for index, entry in enumerate(self.data):
-            if entry[key] == 1:
+            if key in entry and entry[key] == 1:
```

Comment:
I got an error here, not sure if that's OK, or if it would be better to err/log?